src/PersonalSite.Persistence/PersonalSite.Persistence.csproj

```diff
@@ -11,7 +11,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\PersonalSite.Application\PersonalSite.Application.csproj" />
+    <ProjectReference Include="..\PersonalSite.Domain\PersonalSite.Domain.csproj" />
```

Comment:
Just a suggestion: this is not needed as the domain is referenced by the Application, we shoud remove that